### PR TITLE
Admin for posts

### PR DESCRIPTION
**Pull request by mushkin-v**

### CODE REVIEW

File: app/Resources/views/SonataAdmin/base_layout.html.twig [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+{% block stylesheets %}
+    {{ parent() }}
+{% endblock %}
+
+{% block side_bar_after_nav %}
+{% endblock %}
+
+{% block javascripts %}
+
+    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.0/jquery.min.js"></script>
```
Comment (spolischook) [inline app/Resources/views/SonataAdmin/base_layout.html.twig]: It will not work

File: app/config/admin.yml [hunk 2]
```diff
@@ -42,3 +42,25 @@ services:
...
+       tags:
+           - { name: sonata.admin, manager_type: orm, group: "CheTheatre", label: "Posts" }
+       arguments:
+           - ~
+           - AppBundle\Entity\Post
+           - ~
+       calls:
+       - [ setTranslationDomain, [AppBundle]]
+
+   sonata.admin.tag:
```
Comment (spolischook) [inline app/config/admin.yml]: remove this

File: app/config/sonata_admin.yml [hunk 3]
```diff
@@ -2,6 +2,10 @@ sonata_admin:
     title:      %sonata_admin.title%
     title_logo: %sonata_admin.logo_title%
 
+    templates:
+        edit: ::SonataAdmin/CRUD/edit.html.twig
+        layout: :SonataAdmin:base_layout.html.twig
```
Comment (spolischook) [inline app/config/sonata_admin.yml]: we need to redefine only edit template

File: app/config/sonata_admin.yml [hunk 4]
```diff
@@ -24,6 +28,22 @@ sonata_admin:
                 settings:
                     groups: [sonata_media, Hidden]
 
+    assets:
+        javascripts: # changed order of js assets loading
+#            - bundles/sonataadmin/vendor/jquery/dist/jquery.min.js
```
Comment (spolischook) [inline app/config/sonata_admin.yml]: redefine jquery in template

File: src/AppBundle/Admin/TagAdmin.php [hunk 5]
```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace AppBundle\Admin;
+
+use Sonata\AdminBundle\Admin\Admin;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Datagrid\DatagridMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+
+class TagAdmin extends Admin
```
Comment (spolischook) [inline src/AppBundle/Admin/TagAdmin.php]: remove this

File: app/Resources/views/SonataAdmin/base_layout.html.twig (hunk 1 above)
Comment (spolischook) [inline app/Resources/views/SonataAdmin/base_layout.html.twig]: just redefine by config

File: app/config/sonata_admin.yml (hunk 4 above)
Comment (spolischook) [inline app/config/sonata_admin.yml]: define here that version of jquery that you need

File: app/config/sonata_admin.yml (hunk 3 above)
Comment (mushkin-v) [inline app/config/sonata_admin.yml]: There is a problem... I will write about this later down here...

File: app/config/admin.yml (hunk 2 above)
Comment (mushkin-v) [inline app/config/admin.yml]: Remove Tags from sontata - it is Ok

File: src/AppBundle/Admin/TagAdmin.php (hunk 5 above)
Comment (mushkin-v) [inline src/AppBundle/Admin/TagAdmin.php]: Remove Tags from sontata - it is Ok

File: app/Resources/views/SonataAdmin/base_layout.html.twig (hunk 1 above)
Comment (mushkin-v) [inline app/Resources/views/SonataAdmin/base_layout.html.twig]: There is a problem... It dont work from config... I dont understand why, but when I try to do it in config its fail...

File: app/config/sonata_admin.yml (hunk 4 above)
Comment (mushkin-v) [inline app/config/sonata_admin.yml]: I try to do this... but it fails... It seems strange to me, but it dont work...

Comment (mushkin-v) [inline app/config/sonata_admin.yml]: Its only work when I add it in base_layout.html.twig...

File: app/Resources/SonataAdminBundle/views/CRUD/base_list_field.html.twig [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+        {% set isEditable = field_description.options.editable is defined and field_description.options.editable and admin.isGranted('EDIT', object) %}
+        {% set xEditableType = field_description.type|sonata_xeditable_type %}
+
+        {% if isEditable and xEditableType %}
+            {% set url = path('sonata_admin_set_object_field_value', { 'context': 'list', 'field': field_description.name, 'objectId': admin.id(object), 'code': admin.code(object) })  %}
+            <span {% block field_span_attributes %}class="x-editable" data-type="{{ xEditableType }}" data-value="{{ value }}" data-title="{{ field_description.label|trans({}, field_description.translationDomain) }}" data-pk="{{ admin.id(object) }}" data-url="{{ url }}" {% endblock %}>
+                {{ block('field') }}
+            </span>
+        {% else %}
+            {% if field_description.name=='avatar' or field_description.name=='mainPicture' %}
```
Comment (spolischook) [inline app/Resources/SonataAdminBundle/views/CRUD/base_list_field.html.twig]: and if we add one more field this condition will not work

File: app/Resources/SonataAdminBundle/views/Core/user_block.html.twig [hunk 7]
```diff
@@ -0,0 +1,7 @@
+{% block user_block %}
+    <br/>
+    <a href="{{ path('sonata_admin_dashboard') }}">&nbsp;&nbsp;Main page</a>
+    <br/><br/>
+    <a href="{{ path('nelmio_api_doc_index') }}">&nbsp;&nbsp;Logout</a>
```
Comment (spolischook) [inline app/Resources/SonataAdminBundle/views/Core/user_block.html.twig]: It's not a logout, you need to rename anchor of this link, or make a ticket for update this link

**Comment by spolischook** [comment]
LGTM, expect logout link. Fix it and merge

**Pull request merged**